src/main/java/org/example/daos/SalesEmployeeDao.java

```diff
@@ -0,0 +1,153 @@
+package org.example.daos;
+
+import org.example.models.SalesEmployee;
+import org.example.models.SalesEmployeeRequest;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SalesEmployeeDao {
+    final int parameterOne = 1;
+    final int parameterTwo = 2;
+    final int parameterThree = 3;
+    final int parameterFour = 4;
+    final int parameterFive = 5;
+    final int parameterSix = 6;
```

Comment:
dont like just saying paramn to get out of magic numbers, variables should be descriptive

Reply:
make them static, make them capital